### PR TITLE
upgrade interface version

### PR DESCRIPTION
**Pull request by mikerott**

### CODE REVIEW

File: compression.go [hunk 1]
```diff
@@ -15,7 +15,7 @@ import (
 // runsqs.SQSProducer to produce the compressed message.
 // We must base64 encode the compressed message because SQS
 // supports only character data -- valid utf8 -- not raw binary data
-type CompressionSQSProducer struct {
+type CompressionSQSProducer struct { // nolint
```
Comment (FloopySaucer) [inline compression.go:18]: I'm pretty sure this lint check fail is for having the package name in the struct type. Could we not just rename it to `SQSProducer`

File: compression_test.go [hunk 2]
```diff
@@ -10,7 +10,13 @@ import (
 	"github.com/golang/mock/gomock"
 )
 
-func TestCompression(t *testing.T) {
+const (
+	helloworld               = `hello world!`
+	encodedCompressedMessage = `H4sIAAAAAAAA/8pIzcnJVyjPL8pJUQQEAAD//23CtAMMAAAA`
+	exampleURL               = `www.example.com`
+)
```
Comment (FloopySaucer) [inline compression_test.go:17]: Nice!

File: compression_test.go [hunk 3]
```diff
@@ -10,7 +10,13 @@ import (
 	"github.com/golang/mock/gomock"
 )
 
-func TestCompression(t *testing.T) {
+const (
+	helloworld               = `hello world!`
```
Comment (FloopySaucer) [inline compression_test.go]: Nit: Use camel case. 
```suggestion
	helloWorld               = `hello world!`
```

Change would need to be propagated elsewhere too, but can't do the suggestion bit in here :(

Comment (mikerott) [reply compression_test.go]: haha.  Ok FINE.  :)

File: compression.go (hunk 1 above)
Comment (mikerott) [reply compression.go:18]: yeah I was going to "least changes" in this PR so I didn't have to release a major version change due to broken signature.  I'm ok either way.  Opinion?

Comment (FloopySaucer) [reply compression.go:18]: That's completely fair. Considering this is a pretty small library and barely touched, I'm good with keeping it as is 👍

**Review by FloopySaucer — APPROVED** [review]

**Review by ItsMeWithTheFace — APPROVED** [review]

**Pull request merged**